.agents/skills/wayfinder-development/SKILL.md

```
@@ -0,0 +1,87 @@
+- Creating links or navigation to backend endpoints
+
+## Documentation
+
+Use `search-docs` for detailed Wayfinder patterns and documentation.
+
+## Quick Reference
+
+### Generate Routes
+
+Run after route changes if Vite plugin isn't installed:
+```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)